ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/svarinn/SvarInnMessage.java

```diff
@@ -20,7 +20,13 @@
     @NonNull
     List<SvarInnFile> svarInnFiles;
 
-    private PayloadConverter<MeldingType> payloadConverter = new PayloadConverterImpl<>(MeldingType.class, "http://www.arkivverket.no/Noark4-1-WS-WD/types", MeldingType.class.getName());
+    private PayloadConverter<MeldingType> payloadConverter = new PayloadConverterImpl<>(MeldingType.class,
+            "http://www.arkivverket.no/Noark4-1-WS-WD/types", "Melding");
+
+    public SvarInnMessage(Forsendelse forsendelse, List<SvarInnFile> svarInnFiles) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) "svarInnFiles" is marked "lombok.NonNull" but is not initialized in this constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS2637)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) "forsendelse" is marked "lombok.NonNull" but is not initialized in this constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS2637)